src/parser.cpp

```diff
@@ -173,9 +176,9 @@ struct QueryGrammar : qi::grammar<Iterator, Command(), Skipper>
   qi::rule<Iterator, Skipper> create_index;
 
   qi::rule<Iterator, InsertIntoCommand(), Skipper> insert_into;
+  qi::rule<Iterator, UpdateTableCommand(), Skipper> update;
```

Comment:
This one is a bit nitpick, can you name the command to `UpdateCommand` so that it can be consistent with the rule name?